addons/medical_engine/CfgActions.hpp

```diff
@@ -13,4 +13,32 @@ class CfgActions {
     class FirstAid: None {
         show = 0;
     };
+    class UnloadUnconsciousUnits: None
+    {
```

Comment:
!!!

Reply:
Haha, that's what I get for taking it out of vanilla configs